tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/TupleAccessExprTest.java

```diff
@@ -38,10 +41,12 @@
 public class TupleAccessExprTest {
 
     private CompileResult compileResult;
+    private CompileResult compileNegativeResult;
 
     @BeforeClass
     public void setup() {
         compileResult = BCompileUtil.compile("test-src/types/tuples/tuple_access_expr.bal");
+        compileNegativeResult = BCompileUtil.compile("test-src/types/tuples/tuple_with_no_member_test.bal");
```

Comment:
Rename the test file as https://github.com/ballerina-platform/ballerina-lang/pull/37908/files#r981893916